server/src/watchdog.js

```diff
@@ -240,10 +240,12 @@ exports.handleResult = function(req) {
 };
 
 function handleNegative(record) {
+  mozlog.debug("watchdog-negative-match", {msg: `Watchdog submission ${record.id} is a negative match.`});
   return db.update("UPDATE watchdog_submissions SET positive_result = FALSE WHERE id = $1", [record.id]);
 }
 
 function handlePositive(record) {
+  mozlog.warn("watchdog-positive-match", {msg: `Watchdog submission ${record.id} is a positive match.`});
```

Comment:
Although a positive match is important, it's not actually an error state, so use `info` instead of `warn` here